tools/topology/topology2/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+# SPDX-License-Identifier: BSD-3-Clause
+add_custom_target(topologies2 ALL)
+
+add_dependencies(topologies2 topologies1)
+
+# check alsatplg version and return if version is less than 1.2.6
+execute_process(COMMAND alsatplg --version OUTPUT_VARIABLE ALSATPLG_VERSION OUTPUT_STRIP_TRAILING_WHITESPACE)
+string(REGEX MATCH "[0-9].[0-9].[0-9]" VERSION_NUMBER ${ALSATPLG_VERSION})
```

Comment:
Should you make the MINOR optional?

Reply:
no, we want the 1.2.6 version which has the -I option added.

Reply:
i don;t understand. we check if version is less than 1.2.6, why will 1.3 be not supported?

Reply:
the string command doesnt check for version number. it only extracts it here

Reply:
I believe `REGEX MATCH "[0-9].[0-9].[0-9]` will fail to extract 1.3

Reply:
you're right. fixed now